comment/src/comment.ts

```diff
@@ -0,0 +1,280 @@
+import { Text, Line } from "@codemirror/next/text"
+import { EditorView, Command } from "@codemirror/next/view"
+import { EditorState, Transaction, SelectionRange, Change } from "@codemirror/next/state"
+
+/// Comments or uncomments the current `SelectionRange` using line-comments.
+/// The line-comment token is defined on a language basis.
+export const toggleLineCommentCmd: Command = view => {
```

Comment:
I'd prefer a naming convention without the `Cmd` suffix (other packages do the same—the command to undo is simply called `undo`)

Reply:
If you make the commands take a `{state, dispatch}` object instead of an editor view, you can run the tests without exporting the helper functions, making the interface a bit simpler (and the docs shorter)

Reply:
Good to know. I wasn't aware of the the StateCommand. Thanks for the tip.